ledger/participant-integration-api/src/main/scala/platform/indexer/sequential/SequentialIndexer.scala

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.indexer.sequential
+
+import java.sql.Connection
+
+import com.daml.ledger.participant.state.v1.{Offset, Update}
+import com.daml.platform.store.appendonlydao.JdbcLedgerDao
+import com.daml.platform.store.backend.{DBDTOV1, StorageBackend}
+
+import scala.util.chaining.scalaUtilChainingOps
+
+trait SequentialIndexer {
```

Comment:
Will this class also be used for sandbox-on-x?

Reply:
AFAIK not: with sandbox-on-x we do not need sequential inserts, we still want to do parallel ingestion if possible.
sandbox-on-x will have some groovy conflict checking logic otherwise to take care of conflicts.